Resources/doc/annotations/view.rst

```diff
@@ -57,14 +57,19 @@ case for the above example, you can even omit the annotation value::
         return array('post' => $post);
     }
 
+.. tip::
+   Sub namspaces are converted into underscores. 
+   A ``UserProfileController`` with a ``showDetailsAction`` 
```

Comment:
Maybe use the FQCN?
`Sensio\BlogBundle\Controller\UserProfileController::showDetailsAction()`

Reply:
Okay, good idea!